docs/DafnyRef/Programs.md

```diff
@@ -40,27 +40,33 @@ a file that will be included in the Dafny source. These included
 files also obey the ``Dafny`` grammar. Dafny parses and processes the
 transitive closure of the original source files and all the included files,
 but will not invoke the verifier on the included files unless they have been listed
-explicitly on the command line.
+explicitly on the command line or the `--verify-included-files` option is
+specified.
 
-The file name may be a path using the customary `/`, `.`, and `..` specifiers.
+The file name may be a path using the customary `/`, `.`, and `..` punctuation.
 The interpretation of the name (e.g., case-sensitivity) will depend on the
 underlying operating system. A path not beginning with `/` is looked up in
-the underlying file system relative to the current working directory (the
-one in which the dafny tool is invoked). Paths beginning with a device
+the underlying file system relative to the _location of the file in which the
+include directive is stated_. Paths beginning with a device
```

Comment:
```suggestion
include directive is stated. Paths beginning with a device
```

Reply:
The _ is the end of an emphasis. Changed line breaks to make it clearer.

Reply:
Oh, I see it now.